deliveries/bluesky_milestone_1.md

```diff
@@ -0,0 +1,44 @@
+
+**Deliverables**
+> Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which will be used for testing. If you don't provide a commit hash, we will work off the default branch of your repository. Thus, if you plan on continuing work after delivery, we suggest you create a separate branch for either the delivery or your continuing work. 
+> 
+> If there is anything particular about any of the deliverables we or a future reader should know, use the respective `Notes` column.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Define use case | [Food Order on Blockchain](https://medium.com/@opensmartcontract/learn-ink-by-example-order-food-on-blockchain-a4024b2dee4a) | define a contrived use case with [persona](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.s7qu46re6cic), [workflow](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.p52xmq7d7g3m), [target audiences](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.y8aexxoud7oj) for ink! smart contract implementation |
+| 2. | Investigate OpenZepplin's's security / ownership to see how it might fit | [OpenBrush vs openzeppelin on Ownable and AccessControl](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.ahhyroiqktdx) | Implemented ownable for security and restaurants/coutier roles for access control |
+| 3. | Define list of ink! Macros | Define declarative and procedure macros | 1) [Payment_macro](https://github.com/InkSmartContract/foodorder-smartcontract/blob/main/contracts/foodorder/logic/helpers/helpers.rs) as declarative macro; 2) [crud_macro](https://github.com/InkSmartContract/foodorder-smartcontract/tree/main/contracts/foodorder/crud-macro) as procedure macro encapsulating reusable logic to simplify code |
+| 4. | Smart contracts | Implemented in ink! smart contract with macros] | [Github open source repository](https://github.com/InkSmartContract/foodorder-smartcontract.git) |
+| 5. | End-to-end integration | [End-to-end integrate with testing scripts and results](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.ni7dbbifqhne) | [Docker Image](https://hub.docker.com/repository/docker/fpleader/opensmartcontract/) |
+| 6. | Deployment | Deploy smart contract to Shibuya (Shiden testnet) | [Step-by-step Contract Deployment and Interaction Guide](https://docs.google.com/document/d/1stF4dCXdT0fjPur23OD-eTZ90FdOq7tU8x2xEkoPojw/edit#heading=h.iq3vnposvacy) |
```

Comment:
It would be good to link to a faucet in the documentation. make it easier for users if they are just learning about Shibuya